_posts/2020-09-02-hhvm-4.73.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: "HHVM 4.73"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.73 is released! This release marks the end of support for 4.66;
+HHVM 4.67&ndash;4.72 remain supported, as do the 4.32 and 4.56 LTS releases.
+
+# Highlights
+
+- Support for using XHP classes (elements) with namespaces is now enabled by
```

Comment:
This should probably also mention the `xhp class` syntax change, and lack of support for `category`

Reply:
I was planning to make sure it's all listed on the documentation page we link to, but I guess it makes sense to include a short list here too.

Reply:
oh, also the mangling change separately goes in breaking changes, especially re serialization and stored class names

you can still do `class :foo:bar {}`, but:
- it defines `foo\bar` now instead of `xhp_foo__bar`
- you can't do `class :foo:bar-baz {}`